src/tooling_webserver.nim

```diff
@@ -40,12 +41,12 @@ router routes:
     let response = %*
         {
             "exit_status": exit_status,
-            "result": nil
+            "results": %*{}
         }
 
-    let results_filepath = request.params["results_filepath"]
-    if fileExists(fmt"{output_dir}/{results_filepath}"):
-        response["result"] = % readFile(fmt"{output_dir}/{results_filepath}")
+    for results_filepath in request.params["results_filepaths"].split(','):
```

Comment:
As far as I could tell by checking the Jester docs, the `request.params` property is a mapping from strings to strings. This means that I couldn't pass in the result file paths as an array, so I concatenated them using the `,` character.